pychromecast/socket_client.py

```diff
@@ -212,6 +217,21 @@ def __init__(self, host, port=None, cast_type=CAST_TYPE_CHROMECAST,
                                  NetworkAddress(self.host, self.port)))
             raise
 
+    def _get_host_from_service(self, service):
+        """ Resolve host and port from service. """
+        host = self.host
+        port = self.port
+        if service:
+            host, port = get_host_from_service(service, self.zconf)
+            if host and port:
+                self.logger.debug("[%s:%s] Resolved service %s to %s:%s",
+                                  self.host, self.port, service, self.host,
+                                  self.port)
+            else:
+                self.logger.debug("[%s:%s] failed to resolve service %s",
```

Comment:
Should we do something besides debug logging that we failed to resolve?

Reply:
Good point.